data/json/items/gun/380.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "id": "mac_11",
+    "copy-from": "mac_10",
+    "type": "GUN",
+    "name": "MAC-11",
+    "description": "A lesser known variant of the MAC-10, this machine pistol is chambered in .380 ACP for a smaller overall size while remaining inherently subsonic.  Smaller in almost every dimension, this inexpensive automatic weapon was declared 'fit only for combat in a phone booth' due to its low weight and absurd fire rate ranging from 1200 to 1400 rounds per minute.",
+    "weight": 1590,
+    "volume": 2,
+    "price": 160000,
+    "bashing": 5,
+    "ammo": "380",
+    "burst": 23
+  },
+  {
+    "id": "kp3at",
+    "copy-from": "kp32",
+    "type": "GUN",
+    "name": "Kel-Tec P3AT",
+    "description": "Essentially a slightly scaled up Kel-tec P32 in .380 ACP, the ever popular P3AT offers better ballistics in a small, concealable lightweight package.  Handling leaves something to be desired due to snappier recoil and diminuitive controls.",
+    "weight": 240,
+    "volume": "231ml",
+    "price": 25000,
+    "ammo": "380"
+  },
+  {
+    "id": "fn1910",
+    "copy-from": "pistol_backup",
+    "type": "GUN",
+    "name": "FN 1910 .380",
```

Comment:
This little guy should be a VERY rare find considering its production was stopped at 1983.

Reply:
@Night-Pryanik absolutely, that's the idea. Mansions only, basically, and with a tiny item group chance too.

Reply:
I think one variant is enough. Choose the one you feel more fit.